express-intro/server.ts

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+
+const app = express();
+
+app.listen(8080, () => {
+  console.log('Express server listening on port 8080');
+});
+
+app.use((req, res, next) => {
+  console.log('Hello, World!');
+  console.log('The date is', new Date());
+  next();
+});
```

Comment:
Hey Nate, make sure to move this before the `listen` method call. 

the listen method should be at the end of the server

Reply:
![Screenshot 2024-04-04 121246](https://github.com/NateNobleza/rc0124-code-solutions/assets/154506686/83bdfee3-e963-4f50-ae55-9aa90eb9e694)


the listen function is now at the end